src/lang/locales/de-DE.js

```diff
@@ -124,17 +215,20 @@ export default {
     102: "Aufgrund von Inaktivität abgemeldet",
     103: "Benutzer inaktiv",
     106: "Falscher Benutzername und/oder Passwort"
```

Comment:
Oh and there's a `,` missing after this line 🙂 

```suggestion
    106: "Falscher Benutzername und/oder Passwort",
```